conf/route_control.py

```diff
@@ -336,20 +337,24 @@ def __init__(
 
         self._lock = Lock()
 
-        self._ipdb = ipdb
+        self._ndb = ndb
         self._ipr = ipr
         self._bess_controller = bess_controller
         self._ping_missing_thread = Thread(
             target=self._ping_missing_entries, daemon=True
         )
-        self._event_callback = None
         self._interfaces = interfaces
 
-    def register_callbacks(self) -> None:
-        """Register callback function."""
-        logger.info("Registering netlink event listener callback...")
-        self._event_callback = self._ipdb.register_callback(
-            self._netlink_event_listener
+    def register_handlers(self) -> None:
+        """Register handler functions."""
+        logger.info("Registering netlink event listener handlers...")
```

Comment:
I'd encourage to log after the action was completed. Ex. "Registered netlink event listeners" after the `register_handler` calls are made.

Reply:
Done